snap/info_test.go

```diff
@@ -128,6 +128,10 @@ func (s *infoSuite) TestReadInfo(c *C) {
 }
 
 func makeTestSnap(c *C, yaml string) string {
```

Comment:
Ah, no you were right @pedronis, `MakeTestSnapWithFiles` validates the snap, whereas some of these tests test the validation (and thus make invalid snaps), so it can't be used here. It probably makes sense to refactor `MakeTestSnapWithFiles` to allow for this, but that's probably outside the scope of this PR.